components/script/dom/xrsession.rs

```diff
@@ -160,17 +232,18 @@ impl XRSession {
 
         // If the canvas element is attached to the DOM, it is now dirty,
         // and we need to trigger a reflow.
-        if let Some(webgl_layer) = base_layer.downcast::<XRWebGLLayer>() {
-            webgl_layer
-                .Context()
-                .Canvas()
-                .upcast::<Node>()
-                .dirty(NodeDamage::OtherNodeDamage);
-        }
+        base_layer
+            .Context()
+            .Canvas()
+            .upcast::<Node>()
+            .dirty(NodeDamage::OtherNodeDamage);
```

Comment:
This is much nicer now we don't have `XRLayer`!

Reply:
yay